builtin/client/death_formspec.lua

```diff
@@ -4,12 +4,12 @@
 core.register_on_death(function()
 	core.display_chat_message("You died.")
 	local formspec = "size[11,5.5]bgcolor[#320000b4;true]" ..
-		"label[4.85,1.35;" .. fgettext("You died") ..
-		"]button_exit[4,3;3,0.5;btn_respawn;".. fgettext("Respawn") .."]"
+			"label[4.85,1.35;" .. fgettext("You died") ..
+			"]button_exit[4,3;3,0.5;btn_respawn;" .. fgettext("Respawn") .. "]"
```

Comment:
The formatter should not correct this. Both, one indents and two indents on continued lines are valid.
Two indents is AFAIK only an option due to the limited clang format functionalities.

Reply:
Fixed, thanks